src/Microsoft.Performance.Toolkit.Engine/Engine.cs

```diff
@@ -17,12 +17,17 @@
 using Microsoft.Performance.SDK.Runtime.Extensibility.DataExtensions.Repository;
 using Microsoft.Performance.SDK.Runtime.Extensibility.DataExtensions.Tables;
 
+// TODO::support concurrency in the Engine
```

Comment:
I noticed a lot of todos on this PR. Do we want these things for 1.0 or later? Also maybe we can tag/create the todos with the work item?